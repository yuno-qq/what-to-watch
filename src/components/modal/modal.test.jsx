import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"

import Modal from "./modal.jsx"


describe(`<Modal> snapshots:`, () => {
  test(`should render correctly`, () => {
    const tree = renderer.create(<Modal title={`Заголовок`} text={`Текст попапа`} />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
