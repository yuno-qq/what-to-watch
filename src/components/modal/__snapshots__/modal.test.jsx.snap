// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal> snapshots: should render correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal__content"
  >
    <h2>
      Заголовок
    </h2>
    <p>
      Текст попапа
    </p>
  </div>
</div>
`;
